src/java.desktop/windows/classes/sun/awt/windows/WToolkit.java

```diff
@@ -225,16 +217,12 @@ public WToolkit() {
         AWTAutoShutdown.notifyToolkitThreadBusy();
 
         // Find a root TG and attach toolkit thread to it
-        ThreadGroup rootTG = AccessController.doPrivileged(
-                (PrivilegedAction<ThreadGroup>) ThreadGroupUtils::getRootThreadGroup);
+        ThreadGroup rootTG = ThreadGroupUtils.getRootThreadGroup();
```

Comment:
Is the `@SuppressWarnings("removal")` on this constructor still needed?

Reply:
removed